release.yaml

```diff
@@ -10,3 +10,7 @@ rancher-eks-operator-crd:
   - 102.0.2+up1.2.0
   - 102.0.1+up1.1.5
   - 102.0.0+up1.2.0
+neuvector:
```

Comment:
Hi @selvamt94
Apparently the indentation here is wrong, could you fix that?

Reply:
I have updated it. Thanks.